### DISCUSSION TITLE
Problem with DSDT patching X299

### DISCUSSION THREAD

**Discussion by timido22** [xCuri0/ReBarUEFI#180] [Q&A] [answered]
I have the original bios
[X29STLG1.50L.zip](https://github.com/xCuri0/ReBarUEFI/files/14573951/X29STLG1.50L.zip)
after I add the module and have a problem with patching through UEFIPatch.

[X29STLG1.50L -patched.zip](https://github.com/xCuri0/ReBarUEFI/files/14574025/X29STLG1.50L.-patched.zip)

I try to use the DSDT Method. But I found only this 
![image](https://github.com/xCuri0/ReBarUEFI/assets/154282702/2f862c22-2df2-4cc6-9185-fec37da12de9)
[AmiBoardInfo.zip](https://github.com/xCuri0/ReBarUEFI/files/14574031/AmiBoardInfo.zip)
and after I see the error 
![image](https://github.com/xCuri0/ReBarUEFI/assets/154282702/b446a4f5-32ec-4fcc-8f4c-9c202b2f9dad)

Can you help pls?

**Comment by xCuri0** [comment, marked as answer]
no one said to DSDT patch X299
